### ISSUE TITLE
PaddleClas蒸馏训练时，不收敛

### ISSUE THREAD

**Issue by ChrisMengxl** [PaddlePaddle/PaddleClas#1677]
欢迎您使用PaddleClas并反馈相关问题，非常感谢您对PaddleClas的贡献！
提出issue时，辛苦您提供以下信息，方便我们快速定位问题并及时有效地解决您的问题：
 1. PaddleClas版本以及PaddlePaddle版本：请您提供您使用的版本号或分支信息，如**PaddleClas release/2.2**和PaddlePaddle 2.1.0
 2. 涉及的其他产品使用的版本号：如您在使用PaddleClas的同时还在使用其他产品，如PaddleServing、PaddleInference等，请您提供其版本号
 3. 训练环境信息：
  a. 具体操作系统，如**Linux**/Windows/MacOS
  b. Python版本号，如Python3.6/**7**/8
  c. CUDA/cuDNN版本， 如CUDA10.2/cuDNN 7.6.5等
 4. 完整的代码(相比于repo中代码，有改动的地方)、详细的错误信息及相关log
————————————————————————————————————————————
参考 [30 分钟玩转 PaddleClas（进阶版）](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.3/docs/zh_CN/quick_start/quick_start_classification_professional.md)
完成 蒸馏 所有准备，并且运行没有报错，但是：
[2022/02/10 00:50:51] root INFO: [Train][Epoch 10/200][Iter: 30/48]lr: 1.29818, top1: 0.00000, top5: 0.00000, CELoss_Student_Teacher: 6.90515, loss: 6.90515, batch_cost: 0.46951s, reader_cost: 0.00038, ips: 545.25154 images/sec, eta: 1:11:30
[2022/02/10 00:50:56] root INFO: [Train][Epoch 10/200][Iter: 40/48]lr: 1.29801, top1: 0.00000, top5: 0.00000, CELoss_Student_Teacher: 6.90515, loss: 6.90515, batch_cost: 0.46928s, reader_cost: 0.00041, ips: 545.51759 images/sec, eta: 1:11:23
[2022/02/10 00:50:59] root INFO: [Train][Epoch 10/200][Avg]top1: 0.00000, top5: 0.00000, CELoss_Student_Teacher: 6.90514, loss: 6.90514
[2022/02/10 00:51:00] root INFO: [Eval][Epoch 10][Iter: 0/12]CELoss_Student: 6.95024, loss: 6.95024, top1: 0.00000, top5: 0.00000, batch_cost: 1.06017s, reader_cost: 0.66154, ips: 241.47002 images/sec
[2022/02/10 00:51:04] root INFO: [Eval][Epoch 10][Iter: 10/12]CELoss_Student: 6.95057, loss: 6.95057, top1: 0.00000, top5: 0.00000, batch_cost: 0.39536s, reader_cost: 0.00013, ips: 647.51648 images/sec
[2022/02/10 00:51:05] root INFO: [Eval][Epoch 10][Avg]CELoss_Student: 6.95059, loss: 6.95059, top1: 0.00000, top5: 0.00000
[2022/02/10 00:51:05] root INFO: [Eval][Epoch 10][best metric: 0.0]

可能得问题是？配置文件如下：
[resnext152_vd_64x4d_distill_mv3_large_x1_0_d.txt](https://github.com/PaddlePaddle/PaddleClas/files/8034677/resnext152_vd_64x4d_distill_mv3_large_x1_0_d.txt)

**Comment by Intsigstephon** [comment]
我看配置文件里面，你用的是自己的数据集，类别改成了7类；teacher模型 resnext152单独训练试过吗，准确率能到多少？

**Comment by ChrisMengxl** [comment]
> 我看配置文件里面，你用的是自己的数据集，类别改成了7类；teacher模型 resnext152单独训练试过吗，准确率能到多少？

训练过，Top1 约88%

**Comment by Intsigstephon** [comment]
student模型的pretrained改成True试试

**Comment by ChrisMengxl** [comment]
> student模型的pretrained改成True试试

Arch:
  name: "DistillationModel"
  class_num: 7
  pretrained_list:
  freeze_params_list:
  - True
  - False
  models:
    - Teacher:
        name: ResNeXt152_vd_64x4d
        pretrained: "./pretrained/best_model"
        use_ssld: True
    - Student:
        name: MobileNetV3_large_x1_0
        **pretrained: True**

  infer_model_name: "Student"

——结果还是不收敛

**Comment by littletomatodonkey** [comment]
结构需要改成下面这样，添加class_num参数

```
  models:
    - Teacher:
        name: MobileNetV3_large_x1_0
        pretrained: True
        class_num: 7
        use_ssld: True
        dropout_prob: null
    - Student:
        name: MobileNetV3_small_x1_0
        pretrained: False
        class_num: 7
        dropout_prob: null
```

**Comment by ChrisMengxl** [comment]
> dropout_prob: null
十分感谢

**Comment by ChrisMengxl** [comment]
> 结构需要改成下面这样，添加class_num参数
> 
> ```
>   models:
>     - Teacher:
>         name: MobileNetV3_large_x1_0
>         pretrained: True
>         class_num: 7
>         use_ssld: True
>         dropout_prob: null
>     - Student:
>         name: MobileNetV3_small_x1_0
>         pretrained: False
>         class_num: 7
>         dropout_prob: null
> ```

十分感谢。
